src/ignitus-Dashboard/ignitus-Dashboard-Header/Components/dashBoardHeader.js

```diff
@@ -40,64 +53,72 @@ class dashBoardHeader extends React.Component {
     }
     return (
       <React.Fragment>
-        <nav className="navbar navbar-expand-lg whitenav">
+        <nav className="navbar navbar-expand-lg whitenav text-blue">
```

Comment:
Please remove the bootstrap classes and use emotion style CSS with media queries.